### PR TITLE
Assignment-3: ES Features

### PR DESCRIPTION
**Pull request by vaibhav-webonise**
**Problem statement:**  Print numbers from 1 to 10 with specific time intervals between them using ES6 & ES5

### CODE REVIEW

**Review by deepakb-webonise — APPROVED** [review]

File: NumbersInterval.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        for (let index = 1; index <= number; index++) {
+            setTimeout(() => {
+                document.write(`</br> ${index}`);
+            }, index * 1000)
+        }
+    }
+    DisplayNumbers(10);
+</script>
+</body>
+</html>
```
Comment (swapnil-webonise) [inline NumbersInterval.html]: Newline missing at EOF

File: NumbersInterval.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script>
+    const DisplayNumbers = (number) => {
```
Comment (swapnil-webonise) [inline NumbersInterval.html]: the function name should in camelCase

File: NumbersIntervalES5.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <title>Title</title>
+</head>
+<body>
+<script>
+    for (var i = 1; i <= 10; i++) {
+        displayNumbers(i);
+    }
+
+    function displayNumbers(i) {
+        setTimeout(function () {
```
Comment (swapnil-webonise) [inline NumbersIntervalES5.html]: Prefer arrow function

File: NumbersIntervalES5.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            document.write("</br>"+i);
+        }, i * 1000);
+    }
+</script>
+</body>
+</html>
```
Comment (swapnil-webonise) [inline NumbersIntervalES5.html]: Same here

File: NumbersIntervalES5.html [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (swapnil-webonise) [inline NumbersIntervalES5.html]: Indentation?

File: NumbersIntervalES5.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<script>
+    for (var i = 1; i <= 10; i++) {
```
Comment (swapnil-webonise) [inline NumbersIntervalES5.html]: Prefer let over var

File: NumbersIntervalES5.html [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+        displayNumbers(i);
+    }
+
+    function displayNumbers(i) {
```
Comment (swapnil-webonise) [inline NumbersIntervalES5.html]: Instead of `i`, `no` or `number` looks better

**Review by swapnil-webonise — CHANGES_REQUESTED** [review]

File: NumbersIntervalES5.html (hunk 6 above)
Comment (vaibhav-webonise) [reply NumbersIntervalES5.html]: This JS code is done using ES5 features thats why let is not used there as it was mentioned

**Review by swapnil-webonise — APPROVED** [review]

**Pull request merged**